doc/getting-started/getting-started.rst

```diff
@@ -608,6 +615,29 @@ Install ACRN
    The target system will reboot into the ACRN hypervisor and
    start the Ubuntu Service VM.
 
+#. Confirm that you see the GRUB menu with the "Ubuntu-ACRN Board Inspector" entry. Select
+   it and proceed to booting ACRN. (It may be auto-selected, in which case it
+   will boot with this option automatically in 5 seconds.)
+
+   Note: Maybe there are several same items display, you can check them via pressng `e` to
+       select the item with kernel "5.15.0-53-generic".
```

Comment:
I see 5.15.0-56-generic?

Reply:
When moving the curser to "Ubuntu-ACRN Board Inspector" entry, press the key `e`  then GRUB display will contain kernel information `Linux 5.15.0-53-generic`.
The output shows us that we are running kernel version 5.15.0-53.
Here is what each of these numbers mean:
5 – Kernel version.
15 – Major revision.
0 – Minor revision.
53 – Patch number.

So it's the difference between our kernel versions (both OS are Ubuntu22.04). 
Do you think we should modify the sample file to describe the kernel like this:
  "5.15.*-*-generic"?
Or other opinion? Thanks!

Reply:
Maybe the kernel version upgraded. Have modified it in GSG. Thanks!